client/src/components/TaskManager/TaskManger.jsx

```diff
@@ -0,0 +1,142 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable jsx-a11y/no-autofocus */
+/* eslint-disable jsx-a11y/control-has-associated-label */
+/* eslint-disable guard-for-in */
+/* eslint-disable no-restricted-syntax */
+import React, { useState, useRef } from "react";
+import "./taskmanager.css";
+import { IoIosAddCircleOutline } from "react-icons/io";
+import { RiDeleteBin5Line } from "react-icons/ri";
+
+function TaskManager() {
+  const [currentTab, setCurrentTab] = useState("todo");
+  const [newTaskText, setNewTaskText] = useState("");
+  const [isInputVisible, setInputVisible] = useState(false);
+  const [buttonVisible, setButtonVisible] = useState(true);
+  const [tasks, setTasks] = useState({
+    todo: [
+      { id: 1, text: "Navbar slide urgent issue immediately" },
+      { id: 2, text: "gitpush connection+SignUp page" },
+      { id: 3, text: "Homepage to be done before Friday" },
+    ],
+    process: [
+      { id: 1, text: "building features for Homepage" },
+      { id: 2, text: "gathering all features" },
+      { id: 3, text: "Homepage is presenting on Friday" },
+    ],
+    finish: [
+      { id: 1, text: "Completed WelcomePage, LoginPage, SignupPage" },
+      { id: 2, text: "Completed project review" },
+    ],
+  });
+
+  const taskListRef = useRef(null);
+
+  const scrollToTop = () => {
+    taskListRef.current.scrollTop = 0;
+  };
+
+  function toggleInputVisible() {
+    setInputVisible(!isInputVisible);
+    setButtonVisible(!buttonVisible);
+  }
+
+  function addTask() {
+    if (!newTaskText.trim()) return;
```

Comment:
do not touch create and delete tasks function for the moment, next features

Reply:
done